config/main.json

```diff
@@ -1,12 +1,12 @@
 {
   "chartVersions": {
-    "cert-manager": "v0.14.3",
+    "cert-manager": "v0.16.0",
     "external-dns": "2.20.4",
     "grafana": "5.5.4",
     "loki-stack": "0.32.1",
     "matrixbot": "v1.1.1",
     "nginx-ingress": "1.41.1",
-    "polkadot": "v0.29.0",
+    "polkadot": "v0.30.0",
```

Comment:
ci tests did not failed, but they should have in my opinion, wdyt ? This package has not been published yet

Reply:
Indeed, on CI we have just unit tests, this issue will add integration tests https://github.com/w3f/polkadot-deployer/issues/19 feel free to take it ;)